cosign/README.md

```diff
@@ -0,0 +1,247 @@
+          "integratedTime": 174213348,
+          "logIndex": 18030X718,
+          "logID": "c0d23d6ad406973f9559f3ba2d1ca01f84147d8ff5cb8445c224f98b9591810d"
+        }
+      },
+      "Issuer": "https://github.com/login/oauth",
+      "Subject": "user@example.com"
+    }
+  }
+]
+```
+
+### Cosign Library Key-based Verify
+
+To verify a keyless signed cosign signature, you need to verify the following trust chain:
+
+- Check if the artifact matches what is described in the signature
+- Check if the signature was created using the expected public key
+- The root trust of Fulcio and Rekor is sourced from https://tuf-repo-cdn.sigstore.dev/ (optional)
+- The root trust of https://tuf-repo-cdn.sigstore.dev/ is from the cosign binary (optional)
```

Comment:
Why do we need them in the key-based verification?

Reply:
They are marked as optional but when will they be needed?

Reply:
I am following the behavior of cosign CLI at: https://github.com/sigstore/cosign/blob/main/cmd/cosign/cli/verify/verify.go#L174

Reply:
removed.